docs/content/topics/project-developers/brig.md

```diff
@@ -7,4 +7,127 @@ aliases:
+## The command line interface for interacting with Brigade
+
+The `brig` CLI provides access to the full repertoire of supported user
+interactions in Brigade, whether it's logging into Brigade with `brig login`,
+bootstrapping a new Brigade project with `brig init`, creating events with
+`brig event create` -- the list goes on.
+
+In this doc, we'll go over how to [install brig] and then give a brief overview
+of the [suite of commands] that brig provides.
+
+[install brig]: #install-brig
+[suite of commands]: #suite-of-commands
+
+## Install Brig
+
+Brig can be installed via a few methods. Below we demonstrate how to install
+via the [GoFish](#gofish) package manager and via the
+[GitHub releases](#github-releases) page.
+
+You can also build brig from source; see the [Developers] guide for more info.
+
+[Developers]: /topics/developers
+
+### GoFish
+
+[GoFish] is a handy cross-platform package manager supporting all three major
+operating systems (Windows, MacOS and Linux). Assuming GoFish is [installed and
+initialized] locally, we can use it to install the brig CLI. Here's an example
+on a MacOS or Linux machine:
+
+```console
+gofish install brig@2
+gofish link brig@2
+```
+
+[Gofish]: https://gofi.sh/
+[installed and initialized]: https://gofi.sh/#install
+
+### GitHub Releases
+
+To install brig via Brigade's GitHub [releases] page, copy the appropriate
+binary into a directory on your machine that is included in your PATH
+environment variable.
+
+**linux**
+```bash
+curl -Lo /usr/local/bin/brig https://github.com/brigadecore/brigade/releases/download/v2.0.0-beta.2/brig-linux-amd64
+chmod +x /usr/local/bin/brig
+```
+
+**macos**
+```bash
+curl -Lo /usr/local/bin/brig https://github.com/brigadecore/brigade/releases/download/v2.0.0-beta.2/brig-darwin-amd64
+chmod +x /usr/local/bin/brig
+```
+
+**windows**
+```powershell
+mkdir -force $env:USERPROFILE\bin
+(New-Object Net.WebClient).DownloadFile("https://github.com/brigadecore/brigade/releases/download/v2.0.0-beta.2/brig-windows-amd64.exe", "$ENV:USERPROFILE\bin\brig.exe")
+$env:PATH+=";$env:USERPROFILE\bin"
+```
+
+The script above downloads brig.exe and adds it to your PATH for the current
+session. Add the following line to your [PowerShell Profile] to make the change
+permanent.
+
+```powershell
+$env:PATH+=";$env:USERPROFILE\bin"
+```
+
+[releases]: https://github.com/brigadecore/brigade/releases
+[PowerShell Profile]: https://www.howtogeek.com/126469/how-to-create-a-powershell-profile/
+
+## Suite of Commands
+
+To view the full suite of commands that brig supports, simply type `brig` in
+your console. You should see the commands available under `COMMANDS`. These
+include:
+
+  * `event`: Create and manage Brigade [Events]
+  * `init`: Bootstrap a new Brigade [Project]
+  * `login`: Log in to Brigade
+  * `logout`: Log out of Brigade
+  * `project`: Create and manage Brigade [Projects]
+  * `role`: Create and manage system roles for [users] or [service accounts]
```

Comment:
Is this what the help says? (If so, it's a separate issue that needs fixing.) Since roles are predefined, the "create" part is inaccurate. The roles commands are just for managing role assignments.

Reply:
You're right -- this is my mistake.  Will update!